### DISCUSSION TITLE
Working with DJGPP?

### DISCUSSION THREAD

**Discussion by gphg** [edubart/nelua-lang#201] [Q&A] [answered]
Any idea how to get nelua working with [DJGPP](http://www.delorie.com/djgpp/)?

**Comment by edubart** [comment, marked as answer]
Nelua should work with DJGPP, as I've tested this before. In short you have to compile a Nelua program using a DJGPP compiler from your host with a command like:
```
nelua --cc=i686-pc-msdosdjgpp-gcc --cflags="-march=i386 -lemu" --binary --output hello.exe examples/helloworld.nelua
```

Then copy `hello.exe` to the machine you want to run.

Also see https://github.com/edubart/nelua-lang/issues/119 for more details.

**Reply by gphg** [reply]
I read the referred issue. It answers my question. Thanks.
